app/src/helpers/helpers.ts

```diff
@@ -87,3 +87,8 @@ export function getCounterValue(title: string): string {
   const match = itemCountRegex.exec(title);
   return match ? match[1] : undefined;
 }
+
+export function isArgFormatValid(arg: string): boolean {
+  // eslint-disable-next-line
+  return arg.match(/^-[a-z]$/i) || arg.match(/^--[a-z]{2,}(-[a-z]{2,})?$/i) ? true : false;
```

Comment:
Hmmmmmm no, `/^--[a-z]{2,}(-[a-z]{2,})?$/` will break arguments with multiple dashes, like [`--ignore-gpu-blacklist`](https://github.com/jiahaog/nativefier/blob/master/docs/api.md#ignore-gpu-blacklist).

Correct me if I'm wrong (and this is exaaaaaactly the kind of case you should add to your unit tests!), but I think that `/^--[a-z-]{2,}?$/` will work well. It may look weird if you're young with regexes because of the dash between a and z, and the "standalone" dash, but the regex parser will handle it. Try it and verify it with an extra test!

Reply:
I only tested for one additional dash... oops, I'll modify that. 
`[a-z-]` does not work because it incorrectly accepts `---name`

Reply:
> `[a-z-]` does not work because it incorrectly accepts `---name`

@sorhtyre good catch. `/^--[a-z][a-z-]{1,}?$/i` maybe, then?

Reply:
Got it: `/^--[a-z]{2,}(-[a-z]{2,})*$/i` will catch 0 or more single dashes inside a long argument.
As for the unit test, what other cases should I test against?

Reply:
> As for the unit test, what other cases should I test against?

Your current cases are good, I'd just add a test that `--ignore-gpu-blacklist` is considered valid. That's what unit tests are good for: as you iterate on a function, you realize corner case, adjust the implementation, and you enrich your test cases. That way, as the function gets more complex, you ensure you're not regressing by breaking stuff on the left as you're fixing stuff on the right.